website/content/docs/ecs/terraform/secure-configuration.mdx

```diff
@@ -136,8 +136,7 @@ resource "aws_secretsmanager_secret_version" "gossip_key" {
 
 ### Enable secure deployment
 
-Add the following configurations to enable secure deployment. The `acl_secret_name_prefix`
-should be the same as the `name_prefix` you provide to the ACL controller module.
+Add the following configurations to enable secure deployment.
```

Comment:
I don't know that it is clear that the following configuration is an example/template. Maybe we should say something like "Add the configurations described in the [Secure configurations table] to secure your deployment. You can copy the following example and make adjustments as necessary."
We would need to add a header to the table to link to.

Reply:
```suggestion
To enable secure deployment, add the following configuration to the task module.
```

Reply:
Adding this change to maintain consistency between main and this branch.